extension/public/popup.js

```diff
@@ -0,0 +1,104 @@
+/********** Enabled Sites **********/ 
+
+// UI references
+const toggleList = document.getElementById("toggleList");
+const redditSwitch = document.getElementById("redditSwitch");
+const imgurSwitch = document.getElementById("imgurSwitch");
+const twitterSwitch = document.getElementById("twitterSwitch");
+const addOtherSites = document.getElementById("addOtherSites");
+const globalSwitch = document.getElementById("globalSwitch");
+
+const globalSwitchHandler = () => {
+    if(globalSwitch.checked == true) {
+        redditSwitch.checked = false;
+        imgurSwitch.checked = false;
+        twitterSwitch.checked = false;
+
+        toggleList.style.pointerEvents = "none";
+        toggleList.style.opacity = "50%";
+    }
+    else {
+        toggleList.style.pointerEvents = "auto";
+        toggleList.style.opacity = "100%";
+    }
+}
+
+globalSwitch.onclick = globalSwitchHandler;
+
+
+/********** Keywords **********/ 
+
+// keywords data
+const allowedWordsData = [
+    "ukraine",
+    "blm",
+    "gas prices"
```

Comment:
The two word phrase in here will cause issues with the detection, each word should only be a single word to improve detection accuracy :+1:

Reply:
Since this is hard-coded data just for display, I think that that should be taken into account a little later, once the extension actually interacts with the database.

Reply:
I mean this is us setting the default wordlist that a user might want.

We’ll definitely end up changing it later but in its current form this won’t work with the backend. Could you please fix it before we merge this?